src/WebJobs.Script.WebHost/DependencyInjection/ServiceProviderExtensions.cs

```diff
@@ -0,0 +1,202 @@
+﻿// TODO: Licensing header, and split these? - these extensions are based on Orchard (https://github.com/OrchardCMS/OrchardCore)
```

Comment:
I think recommendation for other MS projects has been to flow the license header, which should be made out of the following block, and also mention the project in the NOTICE file.

```
BSD 3-Clause License
https://opensource.org/licenses/BSD-3-Clause

Copyright (c) .NET Foundation. All rights reserved.
```

Reply:
hmmm we don't have a NOTICE today - @fabiocav recommendations? I was trying to find DryIoc mentions but don't see past the original files, perhaps we should add/adopt the `NOTICE` pattern if going this route? Happy to do that, but maybe independently of this PR and add to it if we go this route (to make sure we capture anything else in there).